src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -89,6 +90,61 @@
         return emails;
     }
 
+    /**
+     * Generate email to notify course co-owners that the feedback session is opening soon,
+     * in case the feedback session opening info was set wrongly.
+     */
+    public List<EmailWrapper> generateFeedbackSessionOpeningSoonEmails(FeedbackSessionAttributes session) {
+        String template = EmailTemplates.USER_FEEDBACK_SESSION.replace("${status}",
+                FEEDBACK_STATUS_SESSION_OPENING_SOON);
+
+        CourseAttributes course = coursesLogic.getCourse(session.getCourseId());
+
+        // notify only course co-owners, no students
+        List<InstructorAttributes> coOwners = instructorsLogic.getCoOwnersForCourse(session.getCourseId());
+        List<EmailWrapper> emails = new ArrayList<>();
+
+        // todo ??? these parts of the template may not be relevant to co-owners
+        String feedbackAction = FEEDBACK_ACTION_SUBMIT_EDIT_OR_VIEW;
```

Comment:
I think `feedbackAction` is not needed, but not sure about `additionalContactInformation`. Maybe Prof @damithc can help us here?

Reply:
@pPris please post a sample of the current email content and mention which part you are not sure about. Hard to decide by looking at the code alone.

Reply:
![image](https://user-images.githubusercontent.com/49683462/129512441-7f3fb03f-cb10-4e84-9a03-233dfd48d8a6.png)

@damithc The part in blue is the `${feedbackaction}` placeholder of the email template, and the part in red is the `${additionalContactInformation}` placeholder.

The feedback action & the parts about the deadline extension don't seem tailored to the co-owners so I'm not what should be removed completely and what should be replaced.

Reply:
Thanks @pPris 
Perhaps we should omit all these bullet points as none of them are relevant to this type of email? 
But it is good if we can state the precise opening time.
Optionally (less important) we can also give a link for the person to go direct to the session in concern.

Reply:
![image](https://user-images.githubusercontent.com/49683462/129524295-ad4573a8-618f-490b-a2f9-78a891e15166.png)

@damithc How does this look? I've added the start time in both the box and the bullet points, maybe you can decide which looks better?

Reply:
@damithc Alternatively here's the version with the submission link. I will get the reviewers' help later to confirm the link but for now I'll wait till this discussion is resolved to commit my changes

![image](https://user-images.githubusercontent.com/49683462/129524912-35276be1-4729-449e-9180-ac7f44e53c48.png)

Reply:
@pPris I'm OK with both, 2nd one is more useful to the user.
The start time is given inside the box. No need to repeat as a bullet point?
Slight revisions to text:
`Heads up! The following feedback session is due to open soon.`

Reply:
@damithc I've made the changes, does this look good to go?

![image](https://user-images.githubusercontent.com/49683462/129532185-38dc51fd-7604-4d87-870f-c5e5fb797095.png)

Reply:
Yup, I'm OK with this @pPris